app/javascript/dashboard/routes/dashboard/settings/canned/AddCanned.vue

```diff
@@ -125,3 +131,19 @@ export default {
   },
 };
 </script>
+
+<style scoped lang="scss">
+::v-deep .mention--box {
+  border-left: 1px solid var(--color-border);
+  border-right: 1px solid var(--color-border);
+  left: 0;
+  margin: auto;
+  right: 0;
+  top: 10rem !important;
```

Comment:
Please use variables here.
```suggestion
  top: var(--space-mega) !important;
```